bucket/lepton.json

```
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://hackjutsu.com/Lepton/",
+    "description": "A lean code snippet manager based on GitHub Gist",
+    "license": {
```

No need to specify URL when it's valid SPDX identifier
```suggestion
    "license": "MIT",
```